roles/repo_setup/tasks/populate_gating_repo.yml

```diff
@@ -19,11 +20,13 @@
         dest: "{{ cifmw_repo_setup_output }}/gating.repo"
 
     - name: Check for DLRN repo at the destination
+      become: "{{ not cifmw_repo_setup_output.startswith(ansible_user_dir) }}"
       ansible.builtin.stat:
         path: "{{ cifmw_repo_setup_output }}/delorean.repo"
       register: _dlrn_repo
 
     - name: Lower the priority of DLRN repos to allow installation from gating repo
+      become: "{{ not cifmw_repo_setup_output.startswith(ansible_user_dir) }}"
```

Comment:
since the conditional is repeated on 3 consecutive tasks perhaps it can be shared in a block?

Reply:
my bad i misread this it is not conditional :/ adding approve

Reply:
ah well, we can still have become for a block ... :)

Reply:
so anyway ... up to you it could be moved to the block level, since you already have one and then you don't need to repeat it

Reply:
Done!